docs/content/use_cases/items/question_the_docs.md

```diff
@@ -8,44 +16,68 @@ os.environ['OPENAI_API_KEY'] = '<YOUR-OPENAI-API-KEY>'
 
 
 ```python
+import os
 from superduperdb import superduper
 from superduperdb.db.mongodb.query import Collection
-import pymongo
 
-db = pymongo.MongoClient().documents
-db = superduper(db)
+# Uncomment one of the following lines to use a bespoke MongoDB deployment
+# For testing the default connection is to mongomock
+
+mongodb_uri = os.getenv("MONGODB_URI","mongomock://test")
+# mongodb_uri = "mongodb://localhost:27017"
+# mongodb_uri = "mongodb://superduper:superduper@mongodb:27017/documents"
+# mongodb_uri = "mongodb://<user>:<pass>@<mongo_cluster>/<database>"
+# mongodb_uri = "mongodb+srv://<username>:<password>@<atlas_cluster>/<database>"
+
+# Super-Duper your Database!
+from superduperdb import superduper
+db = superduper(mongodb_uri)
 
 collection = Collection('questiondocs')
 ```
 
+In this example we use the internal textual data from the `superduperdb` project's API documentation, with the "meta"-goal of 
+creating a chat-bot to tell us about the project which we are using!
+
+
+```python
+import glob
+glob.glob('../superduperdb/docs/content/docs/*/*.md')
```

Comment:
I wonder if it's to reference files or use GitHub links.

If we reference files, then the notebooks are not standalone and require the whole project to be locally available.

Reply:
How would you do this instead?

Reply:
One solution would be:
1. Check if the `../superduperdb/.../docs/*md` exists locally.
2. If down, download the directory from github into a temporary location (e.g, `/tmp/sddb_docs`.
3. Use the temporary directory.

Reply:
For the downloading you can use this script:

```
https://github.com/SuperDuperDB/superduperdb/blob/main/deploy/images/superduperdb/download-examples.sh
```